test/shim/test_sequencer.py

```diff
@@ -209,6 +213,13 @@ def test_shim_sequencer_wrong_mask_dim(self, nii_fieldmap, nii_anat, nii_mask, s
         with pytest.raises(ValueError, match="Mask image must be in 3d"):
             shim_sequencer(nii_fieldmap, nii_anat, nii_wrong_mask, slices, [sph_coil])
 
+    # def test_shim_sequencer_wrong_units(self, nii_fieldmap, nii_anat, nii_mask, sph_coil, sph_coil2, caplog):
+    #     # Change the name of the units
+    #     sph_coil2.units = "T"
+    #     slices = [(0, 2), (1,)]
+    #     shim_sequencer(nii_fieldmap, nii_anat, nii_mask, slices, [sph_coil, sph_coil2])
+    #     assert "The coils don't have matching units:" in caplog.text
+
```

Comment:
Do you plan to implement this test later ? Otherwise I would just remove it

Reply:
Yes, this will eventually be added